src/route.rs

```diff
@@ -126,6 +126,55 @@ impl Route {
         self
     }
 
+    /// Set the service to be called when this route is hit.
+    ///
+    /// ```
+    /// # use actix_web::{*, dev::*, http::header};
+    /// # use std::task::{Context, Poll};
```

Comment:
Minor nit: nested imports are used above, but not here, which disturbs my perfectionism 😅

Reply:
Where would nested imports be used here?

Reply:
`use std::{task::{Context, Poll}, pin::Pin, future::Future};`